samples/tab-calendar/ts/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-tab-calendar-ts",
+    "source": "officeDev",
+    "title": "Tab calendar in Outlook",
```

Comment:
Interact with the calendar in Outlook from personal tab.

Reply:
Fixed